web.go

```diff
@@ -28,6 +30,7 @@ type WebProvider struct {
 	KeyFile  string `description:"SSL certificate"`
 	ReadOnly bool   `description:"Enable read only API"`
 	server   *Server
+	Auth     *types.Auth `description:"Auth Crendetials for Web Point"`
```

Comment:
Maybe an issue with the StructTag here. 
A flag will be created because of the `description` StructTag.
But I think a custom parser is missing on [`type.Auth`](https://github.com/containous/traefik/blob/master/types/types.go#L188)

And Træfik will probably run into an error if the flag `--web.auth` is called.

Maybe we should remove the StructTag and assuming that TOML file is the only way to use this feat ?
WDYF @emilevauge ?